perception/autoware_tensorrt_rtmdet/src/tensorrt_rtmdet.cpp

```diff
@@ -194,16 +186,10 @@ TrtRTMDet::TrtRTMDet(
   out_masks_h_ = std::make_unique<float[]>(
     batch_size_ * max_detections_ * model_input_width_ * model_input_height_);
 
-  if (use_gpu_preprocess) {
-    use_gpu_preprocess_ = true;
-    image_buf_h_ = nullptr;
-    image_buf_d_ = nullptr;
-  } else {
-    use_gpu_preprocess_ = false;
+  if (use_gpu_preprocess_) {
+      image_buf_h_ = nullptr;
```

Comment:
`image_buf_h_` is `std::unique_ptr`, so we do not need to initialize it by ourselves.
https://github.com/autowarefoundation/autoware.universe/blob/fb6fd8d82a365ee12157670dce8db3e7ba93659f/common/cuda_utils/include/cuda_utils/cuda_unique_ptr.hpp#L60

Reply:
Thanks, initialization with `nullptr` is removed

related commit: https://github.com/autowarefoundation/autoware.universe/pull/8165/commits/24fd9af454f4f3ae55a0e87a2f3c111824113fdc